packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -84,7 +84,7 @@ module.exports = ctx => ({
       };
       try {document.addEventListener("DOMContentLoaded", $buo_f,false)}
       catch(e){window.attachEvent("onload", $buo_f)}
-    `]
+    `]*/
```

Comment:
Any reason not to totally delete these lines?

Reply:
None that I can think of. Thanks!